OpenStreetMap/R/osm.R

```diff
@@ -132,25 +132,18 @@ plot.osmtile <- function(x, y=NULL, add=TRUE, raster=TRUE, ...){
 #'		c(33.22949814144951,133.9892578125),
 #'		minNumTiles=4)
 #'autoplot(map)
-#' 
+#'
 #' }
-openmap <- function(upperLeft,lowerRight,zoom=NULL,type=c("osm","osm-bw","maptoolkit-topo",
-				"waze","bing","stamen-toner","stamen-terrain"
-						,"stamen-watercolor","osm-german","osm-wanderreitkarte","mapbox",
-						"esri","esri-topo","nps","apple-iphoto","skobbler",
-						"hillshade","opencyclemap","osm-transport","osm-public-transport",
-						"osm-bbike","osm-bbike-german"),
-		minNumTiles=9L, mergeTiles=TRUE){
-	type <- type[1]
-	if(substring(type, 1, 4) != "http" && !(type %in% c("osm","osm-bw","maptoolkit-topo",
-				"waze","bing","stamen-toner","stamen-terrain"
-						,"stamen-watercolor","osm-german","osm-wanderreitkarte","mapbox",
-						"esri","esri-topo","nps","apple-iphoto","skobbler",
-						"hillshade","opencyclemap","osm-transport","osm-public-transport",
-						"osm-bbike","osm-bbike-german"))){
-		stop("unknown map type")
-		
-	}
+openmap <- function(upperLeft,lowerRight,zoom=NULL,
+                    type=c("osm","osm-bw","maptoolkit-topo","waze","bing",
+                           "stamen-toner","stamen-terrain","stamen-watercolor",
+                           "osm-german","osm-wanderreitkarte","mapbox",
+                           "esri","esri-topo","nps","apple-iphoto","skobbler",
+                           "hillshade","opencyclemap","osm-transport",
+                           "osm-public-transport","osm-bbike","osm-bbike-german"),
+                    minNumTiles=9L, mergeTiles=TRUE){
+  type = if (substring(type, 1L, 4L) == 'http') type else match.arg(type)
```

Comment:
use <- for assignment and check indentation.

Reply:
Not sure what you mean about indentation; pushing `<-` fix now...

Reply:
Shall I bump the version as well while I'm at it? `0.3.4`?

Reply:
1. Indentation in this package is done with tabs. You are using two spaces, this leads to miss-aligned lines depending on your editor settings. for example if you view the file on github this line doesn't look right.
2. Sure, bump the version.
3. I'm happy to add you as an author, under the understanding that you'll be making other significant contributions.

Reply:
Sure, in fact I'll remove myself from `DESCRIPTION` for now until I put up something non-trivial for a PR. Even then I'll start as `[ctb]` = contributor for the moment...